shuttle/db/db_functions.py

```diff
@@ -89,6 +89,13 @@ def check_if_table_exists(table):
         return True
     return False
 
+def add_user(username, role):
+    try:
+        sql = "INSERT INTO SHUTTLE_USERS (USERNAME, ROLE) VALUES ('{0}', '{1}')".format(username, role)
+        results = query(sql, 'write')
+        return 'success'
+    except Exception as error:
+        return 'Error'
```

Comment:
Return better error messages both from this message and to the user. The user can't add a user that is already in the db is a case that isn't covered

Reply:
ie give a more specific error than this 
![Screen Shot 2020-07-15 at 1 05 49 PM](https://user-images.githubusercontent.com/26125143/87579699-f4f2ba00-c69b-11ea-889b-1a0a15f425ca.png)

Reply:
When a user attempts to add a user that is already in the database, it technically doesn't error because the db just ignores it. But you're right a user would be confused

Reply:
I think it would be better to use more specific error messages since although you are just ignoring the error in the try except you could catch it and then return a specific error that gives more information on what actually went wrong.

Also you should disallow the ability to add a new user without selecting a role. Right now if you don't select a role it defaults to Driver but we probably don't want that to happen by default.

Reply:
Ohh gotcha, I didn't realize having no roles was an option, so I defaulted to the one with less permissions. Felt like the safe option.

Reply:
Having no roles isn't an option, but if they don't select anything and just try to add a user make it so they have to choose a role.